.github/pull_request_template.md

```diff
@@ -10,5 +10,5 @@
 - [ ] Jest frontend tests
 - [ ] Cypress end-to-end tests
 - [ ] Migrations are safe to run at scale (e.g. PostHog Cloud) – present proof if not obvious
-- [ ] Frontend/CSS is usable at 320px (iPhone SE) and decent at 360px (most phones)
-- [ ] Breaking changes are backwards-compatible. Ensure old/new frontend requests work with new/old backends, and vice versa.
+- [ ] New/changed UI looks good on mobile (screen width 360 px)
+- [ ] Changes are backwards-compatible across frontend AND backend
```

Comment:
Even though the original line was overly verbose, this condensed version loses meaning as well. The real point here is that rolling updates shouldn't cause downtime due to frontend/backend inconsistencies, yet I don't clearly read that from this line.

And to be really fair, this was added when we suspected something in the deployment system to cause the downtime with moving funnel queries from GET to POST, but it was actually caused by a postgres vs clickhouse difference. So maybe we don't even need this.

Thoughts on how to improve?

Reply:
I think this point doesn't really matter (especially given it was added because of one issue, which turned out to actually be something else) _and_ kind of awkward to enforce. And I agree we shouldn't rip out parts of the API willy nilly, but we don't actually need backwards compatibility in many cases. Hence I just removed the line. WDYT?